modules/tinymce/src/themes/mobile/main/ts/features/Features.ts

```diff
@@ -20,6 +20,7 @@ import * as FontSizeSlider from '../ui/FontSizeSlider';
 import * as ImagePicker from '../ui/ImagePicker';
 import * as LinkButton from '../ui/LinkButton';
 import * as StyleFormats from '../util/StyleFormats';
+import { MobileRealm } from 'tinymce/themes/mobile/ui/IosRealm';
```

Comment:
This should be a relative import path, not an alias import path since it's in the current theme.

- [ ] Fix import path